core/sql/exp/exp_function.cpp

```diff
@@ -40,6 +40,10 @@
 
 
 #include <math.h>
+#include <zlib.h>
+#include "openssl/md5.h"
```

Comment:
I'm wondering why this isn't <openssl/md5.h>. Seems like one would have to copy the md5.h file into the source tree somewhere for this to compile cleanly. Maybe you meant to use angle brackets instead of quotes?